### PR TITLE
backend: add term and section for submitting and editing review

### PR DESCRIPTION
**Pull request by chinathaip**
- add term + section on review table, use only if `ENABLE_TERM_SECTION_VALIDATION` env is set to `true`
- generate raw DDL for seeding sandbox database in integration tests
- properly apply database migration

### CODE REVIEW

File: apps/reviews-api/review/data/repository/review_repo_impl.go [hunk 1]
```diff
@@ -107,6 +126,14 @@ func (r *reviewRepositoryImpl) SubmitReview(ctx context.Context, review *db.Revi
 }
 
 func (r *reviewRepositoryImpl) EditReview(ctx context.Context, review *db.ReviewModel, courseCode, userID string) (*db.ReviewModel, error) {
+	if err := isTermValid(review); err != nil {
+		return nil, err
+	}
+
+	if err := isSectionValid(review); err != nil {
+		return nil, err
+	}
```
Comment (Lxkas) [inline apps/reviews-api/review/data/repository/review_repo_impl.go:135]: Where is the `GetBoolEnv` and feature flag for this XD

Comment (chinathaip) [reply apps/reviews-api/review/data/repository/review_repo_impl.go:135]: OH SHOOT GOOD CATCH

**Comment by NayHtetKyaw** [comment]
so we are specifying term and section now

**Comment by chinathaip** [comment]
> so we are specifying term and section now

yeah, I think it should allow the university to use this information for improving the course experience. Also, got a green light from the acting president himself to add this XD

**Review by Lxkas — APPROVED** [review]

**Pull request merged**